account_receipt_print/report/a5_report.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data noupdate="1">
+        <record id="paperformat_euro_a5" model="report.paperformat">
```

Comment:
Where did you take the specifications to till the following fields? Could you please share a link?

Reply:
For margins and spacing I used a4 paperformat as a reference (proportional).
Anyway, I changed my mind about this. I think that a new paperformat should be added in a separate module,  a base one if needed. I have removed it accordingly.